tests/nginx-tests/tengine-tests/ngx_http_upstream_vnswrr.t

```diff
@@ -66,6 +66,20 @@ http {
         server 127.0.0.1:8084 down;
     }
 
+    upstream g {
+        vnswrr;
+        server 127.0.0.1:8081 weight=2;
+        server 127.0.0.1:8082 weight=4;
+        server 127.0.0.1:8083 weight=8;
+    }
+
+    upstream g0 {
+        vnswrr;
+        server 127.0.0.1:8081 weight=0;
+        server 127.0.0.1:8082 weight=0;
+        server 127.0.0.1:8083 weight=0;
```

Comment:
Cant run test case because nginx cannot allow 0 weight. 

case failed with error: # 2022/10/30 13:52:36 [emerg] 5636#0: invalid parameter "weight=0" in /tmp/nginx-test-Wz0fXmRZkK/nginx.conf:63\

Reply:
Thanks, I will remove this case